### PR TITLE
Documentation for several BbS files

### PR DESCRIPTION
**Pull request by Truthkey**
This includes updated documentation.

Updated information:
- PMO
- PMP
- PAM

New file formats:
- EXA
- EPD
- OLO
- BEP
- PTX

### CODE REVIEW

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]
For some reason, tests seem to fail with this PR. And I cannot figure out why.
Can I request that you re-base this PR, just in case?

**Comment by Truthkey** [comment]
Rebased it. Should be fine now.

**Comment by Topaz-Reality** [comment]
The test still fails. I will look deep into why that is.

**Comment by Topaz-Reality** [comment]
Apparently, some files you changed causes the tests to fail. For some reason, the SLN file changed and it's changes regard Imgd and Imgz. Can you look into this as I do not understand why tests fail _exactly_?

File: OpenKh.Bbs/Olo.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            [Data] public short unkStructNumber1 { get; set; } // Pretty much always 5.
+            [Data] public short unkStructNumber2 { get; set; }
+            [Data] public int NumObjectsToSpawn { get; set; }
+            [Data] public int HeaderLength { get; set; }
+            [Data] public List<UnkStruct> UnkData { get; set; } // Repeats by unkStructNumber1.
+            [Data] public int padding { get; set; }
+
+            static Header()
+            {
+                BinaryMapping.SetMemberLengthMapping<Header>(nameof(UnkData), (o, m) => o.unkStructNumber1);
```
Comment (Topaz-Reality) [inline OpenKh.Bbs/Olo.cs:70]: Thanks to @Rikux3 for noticing. ``SetMemberLengthMapping`` was removed from the latest version of Xe.BinaryMapper for being thread-unsafe. Please revise this part of the code.

**Review by Topaz-Reality — CHANGES_REQUESTED** [review]

**Comment by Topaz-Reality** [comment]
Closing this PR as it's author declared that they would create a new PR with all the fixes necessary.

**Pull request closed without merging**